Application/MicroBooter/MicroBooter.cpp

```diff
@@ -3,7 +3,7 @@
 //  <No description>
 //
 //  Microsoft dotNetMF Project
-//  Copyright �2004 Microsoft Corporation
+//  Copyright ©2004 Microsoft Corporation

```

Comment:
@miloush just saved it again with that encoding and seems to have remove that diff. Thanks for pointing that. 👏